k9mail-library/src/main/java/com/fsck/k9/mail/store/webdav/WebDavStore.java

```diff
@@ -47,6 +61,7 @@
  * and email information.
  * </pre>
  */
+@SuppressWarnings("deprecation")
```

Comment:
Do we already have an issue to switch to OkHttp - that's really the only way forward here..